gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -148,28 +146,52 @@ def _handle_failure(self, response):
             'No traceback. See logs for more information on error.')
 
   def validate_table(self):
-    """Runs a simple query against the output table and verifies aggregates."""
+    """Runs queries against the output table and verifies aggregates."""
     client = bigquery.Client(project=self._project)
-    # TODO(bashir2): Create macros for common queries and add the option for
-    # having a list of queries instead of just one.
-    query = self._validation_query.format(TABLE_NAME=self._table_name)
-    query_job = client.query(query)
+    # TODO(bashir2): Create macros for common queries
+    query_formatter = QueryFormatter(self._table_name)
+    for assertion_config in self._assertion_configs:
+      query = query_formatter.format_query(assertion_config['query'])
+      assertion = Assertion(client, query, assertion_config['expected_result'])
+      assertion.run_assertion()
+
+
+class Assertion(object):
+  """Runs a simple query against the output table and verifies aggregates."""
+
+  def __init__(self, client, query, expected_result):
+    self._client = client
+    self._query = query
+    self._expected_result = expected_result
+
+  def run_assertion(self):
+    query_job = self._client.query(self._query)
     assert query_job.state == 'RUNNING'
     iterator = query_job.result(timeout=60)
     rows = list(iterator)
     if len(rows) != 1:
       raise TestCaseFailure('Expected one row in query result, got {}'.format(
           len(rows)))
     row = rows[0]
-    if len(self._expected_query_result) != len(row):
+    if len(self._expected_result) != len(row):
       raise TestCaseFailure(
           'Expected {} columns in the query result, got {}'.format(
-              len(self._expected_query_result), len(row)))
-    for key in self._expected_query_result.keys():
-      if self._expected_query_result[key] != row.get(key):
+              len(self._expected_result), len(row)))
+    for key in self._expected_result.keys():
+      if self._expected_result[key] != row.get(key):
         raise TestCaseFailure(
             'Column {} mismatch: expected {}, got {}'.format(
-                key, self._expected_query_result[key], row.get(key)))
+                key, self._expected_result[key], row.get(key)))
+
+
+class QueryFormatter(object):
+  """Formats a query."""
+
+  def __init__(self, table_name):
+    self._table_name = table_name
+
+  def format_query(self, query):
+    return (' ').join(query).format(TABLE_NAME=self._table_name)
```

Comment:
Please add a method comment with a more detailed description of what formatting does (you can enumerate all replacement logics; for now, it's just replacing TABLE_NAME) and an 'Args' section (mainly to explain why query is actually a list of strings rather than a single string).
See [this](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/transforms/merge_headers.py#L107) for an example.